client-tez/src/main/java/org/apache/tez/common/RssTezConfig.java

```diff
@@ -0,0 +1,241 @@
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_RETRY_INTERVAL_MAX;
+  public static final long RSS_CLIENT_RETRY_INTERVAL_MAX_DEFAULT_VALUE =
+          RssClientConfig.RSS_CLIENT_RETRY_INTERVAL_MAX_DEFAULT_VALUE;
+  public static final String RSS_COORDINATOR_QUORUM =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_COORDINATOR_QUORUM;
+  public static final String RSS_DATA_REPLICA = TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_REPLICA;
+  public static final int RSS_DATA_REPLICA_DEFAULT_VALUE = RssClientConfig.RSS_DATA_REPLICA_DEFAULT_VALUE;
+  public static final String RSS_DATA_REPLICA_WRITE =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_REPLICA_WRITE;
+  public static final int RSS_DATA_REPLICA_WRITE_DEFAULT_VALUE =
+          RssClientConfig.RSS_DATA_REPLICA_WRITE_DEFAULT_VALUE;
+  public static final String RSS_DATA_REPLICA_READ =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_REPLICA_READ;
+  public static final int RSS_DATA_REPLICA_READ_DEFAULT_VALUE =
+          RssClientConfig.RSS_DATA_REPLICA_READ_DEFAULT_VALUE;
+  public static final String RSS_DATA_REPLICA_SKIP_ENABLED =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_REPLICA_SKIP_ENABLED;
+  public static final String RSS_DATA_TRANSFER_POOL_SIZE =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_TRANSFER_POOL_SIZE;
+  public static final int RSS_DATA_TRANSFER_POOL_SIZE_DEFAULT_VALUE =
+          RssClientConfig.RSS_DATA_TRANFER_POOL_SIZE_DEFAULT_VALUE;
+  public static final String RSS_DATA_COMMIT_POOL_SIZE =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_COMMIT_POOL_SIZE;
+  public static final int RSS_DATA_COMMIT_POOL_SIZE_DEFAULT_VALUE =
+          RssClientConfig.RSS_DATA_COMMIT_POOL_SIZE_DEFAULT_VALUE;
+
+  public static final String RSS_CLIENT_SEND_THREAD_NUM =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_CLIENT_SEND_THREAD_NUM;
+  public static final int RSS_CLIENT_DEFAULT_SEND_THREAD_NUM =
+          RssClientConfig.RSS_CLIENT_DEFAULT_SEND_NUM;
+  // public static final String RSS_CLIENT_SEND_THRESHOLD = TEZ_RSS_CONFIG_PREFIX + "rss.client.send.threshold";
+  // public static final double RSS_CLIENT_DEFAULT_SEND_THRESHOLD = 0.2f;
+  public static final boolean RSS_DATA_REPLICA_SKIP_ENABLED_DEFAULT_VALUE =
+          RssClientConfig.RSS_DATA_REPLICA_SKIP_ENABLED_DEFAULT_VALUE;
+  public static final String RSS_HEARTBEAT_INTERVAL =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_HEARTBEAT_INTERVAL;
+  public static final long RSS_HEARTBEAT_INTERVAL_DEFAULT_VALUE =
+          RssClientConfig.RSS_HEARTBEAT_INTERVAL_DEFAULT_VALUE;
+  public static final String RSS_HEARTBEAT_TIMEOUT =
+          TEZ_RSS_CONFIG_PREFIX + RssClientConfig.RSS_HEARTBEAT_TIMEOUT;
+  public static final String RSS_ASSIGNMENT_PREFIX = TEZ_RSS_CONFIG_PREFIX + "rss.assignment.partition.";
+  // public static final String RSS_CLIENT_BATCH_TRIGGER_NUM =
```

Comment:
Could we remove unused comments?

Reply:
ok, we corrected some of it and push the code.